pkg/arvo/sur/group-store.hoon

```diff
@@ -32,13 +32,18 @@
 ::    %remove-group: remove a group from the store
 ::    %expose: unset .hidden flag
 ::
-+$  action
-  $%  [%add-group =resource =policy hidden=?]
++$  action  (build-group-diff create:policy action:policy)
+
++$  diff  (build-group-diff policy diff:policy)
+
+++  build-group-diff
+  |$  [create diff]
```

Comment:
Is there any reason you can't add a `metadata-path` to the policy, and then scry/watch for that to populate info? I don't particularly see anything wrong with even just leaving the title and description in, so we don't have this weird action/diff distinction.

Reply:
see below comment